src/py42/services/detectionlists/departing_employee.py

```diff
@@ -40,11 +44,13 @@ def add(self, user_id, departure_date=None):
         Args:
             user_id (str or int): The Code42 userUid of the user you want to add to the departing \
                 employees list.
-            departure_date (str, optional): Date in yyyy-MM-dd format. Date is treated as UTC. Defaults to None.
+            departure_date (str or datetime instance, optional): Date in yyyy-MM-dd format. Date is treated as UTC. Defaults to None.
```

Comment:
Should mention in the description that it can be a `datetime` instance too:

`Date in yyyy-MM-dd format or instance of `datetime`. Date is treated as UTC. Defaults to None.`

Reply:
Changed.